i3_agenda/event_utils.py

```diff
@@ -0,0 +1,49 @@
+
```

Comment:
it would be better, instead of having a "utils" file, to have an `EventFetcher` class, with the `get_closest` method and `get_events` method

Reply:
Well, I agree that there is a better solution that this "utils" file, but just to understand it better: 

* is_allday could be a Event method
* get_event_time, as commented bellow, and get_closest I see as functions, so I propose to these functions only be moved to the event.py file as separated functions

Reply:
What I meant here is that instead of having the `get_closest` and `get_events` as functions, we could have an object called `EventFetcher` and add those functions as its methods